packages/amd-loader/amd-loader.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+(function() {
+
+function Registry() {
```

Comment:
Does `Registry` need to be a "class" at all? If it's a singleton you can just put the state and functions in the module closure.

Reply:
Also got rid of the `Module` class, from offline discussion. Size is now down to 0.7KB :)